source/features/checks-modal.tsx

```diff
@@ -0,0 +1,38 @@
+import {$} from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import delegate, {DelegateEvent} from 'delegate-it';
```

Comment:
The linter only enforces the order (npm up, local down) and one space below each group (one space under npm, one space under local). The auto fixer adds these extra spaces we should drop

```suggestion
import * as pageDetect from 'github-url-detection';
import delegate, {DelegateEvent} from 'delegate-it';
```